google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/spi/DefaultBigQueryRpc.java

```diff
@@ -80,7 +80,7 @@
   private final Bigquery bigquery;
 
   public DefaultBigQueryRpc(BigQueryOptions options) {
-    HttpTransportOptions transportOptions = options.getHttpTransportOptions();
+    HttpTransportOptions transportOptions = (HttpTransportOptions) options.getTransportOptions();
```

Comment:
Presumably once we add support for GrpcTransportOptions we will have to start checking for null here?

And regarding the naming for `DefaultBigQueryRpc`, does that mean we are going to end up with one class called `DefaultBigQueryRpc` and one called `GrpcBigQueryRpc`? Should we consider renaming to `HttpBigQueryRpc` now?